gdk/static/project_build_system.json

```diff
@@ -20,7 +20,7 @@
             "build"
         ],
         "build_command_win": [
-            "gradlew",
+            "gradle",
```

Comment:
why this change?

Reply:
just `gradle` works fine without using `.bat` wrapper